packages/calling/src/CallHistory/CallHistory.test.ts

```diff
@@ -247,4 +249,42 @@ describe('Call history tests', () => {
       );
     });
   });
+
+  describe('fetchLinesData test', () => {
+    it('verify successful lines API case', async () => {
```

Comment:
Can we add the keyword UCM here. We already have an existing fetchLines so this might cause a confusion in the code.

Reply:
Done